packages/quick_actions/quick_actions/example/lib/main.dart

```diff
@@ -53,14 +53,16 @@ class _MyHomePageState extends State<MyHomePage> {
       const ShortcutItem(
         type: 'action_one',
         localizedTitle: 'Action one',
-        icon: 'AppIcon',
+        localizedSubtitle: 'Action one subtitle',
+        icon: 'icon_help',
```

Comment:
Why was the icon changed here?

Reply:
I changed it back to the previous `AppIcon`.